pages/content/amp-dev/documentation/guides-and-tutorials/integrate/embed-stories-nonamp.md

```diff
@@ -0,0 +1,542 @@
+
+`title` (optional)
+
+The title of your story.
+
+`posterImage` (optional)
+
+The poster image of your story.
+
+
+# Customize Web Story Player UI
+
+You may customize the controls of the Web Story Player UI. You can add new control buttons, and change their position or look. Controls are configured as JSON with `type="application/json"` as a child of the `<amp-story-player>` element. Include an array of desired "controls".
+
+```html
+<amp-story-player>
+
+ <script type="application/json">
+   {
+     "controls": [
+       {
+         "name": "close",
+         "position": "start"
+       },
+       {
+         "name": "skip-next"
+       }
+     ]
+   }
+ </script>
+<a href="./story1.html"> ... </a>
+<a href="./story2.html"> ... </a>
+</amp-story-player>
+```
+
+## Close control
+
+By default, the Web Story Player does not include a close control. You may add a close icon to the Web Story Player by specifying a control object with the “close” name. This is recommended when using the player in implementations that display the player in a lightbox or fullscreen view. 
+
+The “close” control supports the following customizable properties:
+
+*   `position`: “start” or “end”.
+    *   Positions the icon either on the left (“start”) or right (“end”) on LTR languages.
+    *   Positions the icon on either the  left (“end”) or right (“start”) on RTL languages.
+*   `visibility`: “hidden” or “visible” (default).
+    *   Toggles the control’s visibility. If omitted, the default is visible.
+*   `backgroundImageUrl`: string with url or data string (escaped).
+    *   Changes the icon image to the provided url or data string (for inline svgs).
+
+The close button dispatches the `amp-story-player-close` event when clicked/tapped by the user. 
+
+```html
+<amp-story-player>
+ <script type="application/json">
+   {
+     "controls": [
+       {
+         "name": "close",
+         "backgroundImageUrl": "data:image\/svg+xml;charset=utf-8,<\/svg>",
+         "position": "start"
+       }
+     ]
+   }
+ </script>
+<a href="./story1.html"> ... </a>
+<a href="./story2.html"> ... </a>
+</amp-story-player>
+```
+
+{{ image('/static/img/docs/close-button-custom-background.png', 363, 603, layout='intrinsic', alt='close control with custom background' ) }}
+
+## Share control
+
+The share control is visible in all stories, but you may customize the look and position.
+
+The “share” control supports the following customizable properties:
+
+*   `position`: “start” or “end”.
+    *   Positions the icon either on the left (“start”) or right (“end”) on LTR languages.
+    *   Positions the icon on either the  left (“end”) or right (“start”) on RTL languages.
+*   `visibility`: “hidden” or “visible” (default).
+    *   Toggles the control’s visibility. If omitted, the default is visible.
+*   `backgroundImageUrl`: string with url or data string (escaped).
+    *   Changes the icon image to the provided url or data string (for inline svgs).
+
+
+## Skip-next control
```

Comment:
Does "skip next" and "skip to next" have the same meaning in English? This feature is "skip to next"